app/schemas.py

```diff
@@ -58,20 +76,22 @@ def __post_dump(self, data):
                     (str(item) if isinstance(item, UUID) else item)
                     for item in value
                 ]
+        return data
 
 
 class BaseSchema(ma.SQLAlchemyAutoSchema):
 
     class Meta:
         load_instance = True
         include_relationships = True
+        unknown = EXCLUDE
```

Comment:
i kinda wonder about this - do i understand correctly that this means if the admin's passing in a field we haven't explicitly listed, then it gets silently dropped and we never know (eg if the field has a typo). if we remove this, do all the normal endpoints fail really noisily for everything? it'd be a nice goal to get rid of this but i don't think that's a high priority as i suspect that's a lot of work for relatively little gain

Reply:
Yes, this means that fields will get silently dropped if they're not part of the schema. The new marshmallow default behaviour is to raise a `ValidationError` for unknown keys. We could look at switching to that in after this goes in - I haven't really checked how many errors (if any) we'd start getting. Setting this option to `EXCLUDE` means we keep the existing behaviour for now, which seemed safest since there are a lot of other marshmallow changes happening.